docs/index.md

```diff
@@ -189,6 +189,39 @@ This package uses the JSON Web Token Python implementation, [PyJWT](https://gith
 
 This is the secret key used to sign the JWT. Make sure this is safe and not shared or public.
 
+Can be a dict, a list or a scalar.
+
+* When a dict, the dict keys are taken as the JWT key ids and the values as
+  keys, e.g.:
+
+  ```python
+  { "kid1": key1, "kid2": key2, ... }
+  ```
+
+  The first element is used for signing.
+
+  If a JWT to be verified contains a key id (`kid` header), only the
+  key with that id is tried (if any).
+
+* When a list, all elements are accepted for verification and the
+  first element is used for signing.
+
+* When a scalar, this secret is used for signing and verification.
+
+*NOTE: For python < 3.7, use a `collections.OrderedDict` object*, e.g.:
+
+  ```python
+  from collections import OrderedDict
+
+  JWT_AUTH["JWT_SECRET_KEY"] = OrderedDict(kid1=key1, kid2=key2, ...)
+  ```
+
+(The first) `JWT_SECRET_KEY` is only used for signing if (the first)
+`JWT_ALGORITHM` is `HS*`, otherwise `JWT_PRIVATE_KEY` is used.
+
+`JWT_SECRET_KEY`(s) is/are only used for verification of JWTs with
+`alg` matching `HS*`
```

Comment:
What do you say we move this block up under the `When a dict` point, just before the second `When a list` point? That way we'll reduce the risk of users skipping it by accident.

Reply:
makes sense